gtda/time_series/embedding.py

```diff
@@ -513,7 +509,211 @@ def resample(self, y, X=None):
+    >>> MTE = MultiTakensEmbedding(time_delay=1, dimension=2)
+    >>> print(embedder.fit_transform(X))
+    [[[0 1]
+      [1 2]
+      [2 3]]
+
+     [[5 6]
+      [6 7]
+      [7 8]]]
+    # Two multivariate time series of duration 4, with 2 variables
+    >>> x = np.arange(8).reshape(2, 1, 4)
+    >>> X = np.concatenate([x, -x], axis=1)
+    >>> print(X)
+    [[[ 0  1  2  3]
+      [ 0 -1 -2 -3]]
+
+     [[ 4  5  6  7]
+      [-4 -5 -6 -7]]]
+    # Pass `flatten` as `True` (default)
+    >>> MTE = MultiTakensEmbedding(time_delay=1, dimension=2, flatten=True)
+    >>> print(MTE.fit_transform(X))
+    [[[ 0  1  0 -1]
+      [ 1  2 -1 -2]
+      [ 2  3 -2 -3]]
+
+     [[ 4  5 -4 -5]
+      [ 5  6 -5 -6]
+      [ 6  7 -6 -7]]]
+    # Pass `flatten` as `False`
+    >>> MTE = MultiTakensEmbedding(time_delay=1, dimension=2, flatten=False)
+    >>> print(MTE.fit_transform(X))
+    [[[[ 0  1]
+       [ 1  2]
+       [ 2  3]]
+
+      [[ 0 -1]
+       [-1 -2]
+       [-2 -3]]]
+
+
+     [[[ 4  5]
+       [ 5  6]
+       [ 6  7]]
+
+      [[-4 -5]
+       [-5 -6]
+       [-6 -7]]]]
+
+    See also
+    --------
+    TakensEmbedding, SlidingWindow
+
+    Notes
+    -----
+    To compute the Takens embedding of a single univariate time series in the
+    form of a 1D array or column vector, use :class:`TakensEmbedding` instead.
+
+    """
+
+    _hyperparameters = TakensEmbedding._hyperparameters.copy()
+    _hyperparameters.pop('parameters_type')
+    _hyperparameters.update({'flatten': {'type': bool},
+                             'ensure_last_value': {'type': bool}})
+
+    def __init__(self, time_delay=1, dimension=2, stride=1, flatten=True,
+                 ensure_last_value=True):
+        self.time_delay = time_delay
+        self.dimension = dimension
+        self.stride = stride
+        self.flatten = flatten
+        self.ensure_last_value = ensure_last_value
+
+    def fit(self, X, y=None):
+        """Do nothing and return the estimator unchanged.
+
+        This method is here to implement the usual scikit-learn API and hence
+        work in pipelines.
+
+        Parameters
+        ----------
+        X : ndarray or list
+            Input collection of time series. A 2D array or list of 1D arrays is
+            interpreted as a collection of univariate time series. A 3D array
+            or list of 2D arrays is interpreted as a collection of multivariate
+            time series, each with shape ``(n_variables, n_timestamps)``. More
+            generally, :math`N`-dimensional arrays or lists of
+            (:math`N-1`)-dimensional arrays (:math:`N \\geq 3`) are interpreted
+            as collections of tensor-valued time series, each with time indexed
+            by the last axis.
+
+        y : None
+            There is no need for a target, yet the pipeline API requires this
+            parameter.
+
+        Returns
+        -------
+        self : object
+
+        """
+        check_multi_time_series(X, copy=False)
```

Comment:
`check_graphs` checks collection of graphs so `check_time_series` should be the name of the method that checks collection of time series, no?

Reply:
I agree with you! Not sure what I was thinking, I'll change that.

Reply:
Done.